algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala

```diff
@@ -57,8 +57,13 @@ case class Moments(m0: Long, m1: Double, m2: Double, m3: Double, m4: Double) {
 }
 
 object Moments {
-  implicit val group: Group[Moments] with CommutativeGroup[Moments] =
+  @deprecated("use monoid[Moments], this isn't lawful for negate", "0.13.8")
+  def group: Group[Moments] with CommutativeGroup[Moments] =
```

Comment:
we are removing `implicit` but that is not a problem for binary compatibility (and this is checked by mima)